xla_native_functions.yaml

```diff
@@ -316,6 +316,9 @@ supported:
   - ger
   - lerp.Scalar
   - lerp.Tensor
+  - logical_not.out
+  - logical_xor.out
+  - logical_or.out
```

Comment:
Shouldn't we implement `logical_or` instead of `logical_or.out` ([ref link](https://github.com/pytorch/xla/pull/3083#discussion_r689746053))? The same applies to the other two ops.

Reply:
Good point, I will update this